chef/cookbooks/cpe_nightly_reboot/recipes/default.rb

```diff
@@ -0,0 +1,64 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+# Cookbook Name:: cpe_nightly_reboot
+# Recipe:: default
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+require 'Pathname'
+
+# Set up the logout script
+directory Pathname.new(node['cpe_nightly_reboot']['script']).dirname.to_s do
```

Comment:
not runtime safe. 

directory "script path" do
  path lazy { Pathname.new(node['cpe_nightly_reboot']['script']).dirname.to_s }